### PR TITLE
fix(ticket): restores group's tooltip

### PR DESCRIPTION
**Pull request by Rom1-B**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | !33820

In 9.5, on tickets, groups had a tooltip allowing the complete name and comment to be displayed, but this was no longer present on 10.0.

Before:
image

After:
image

### CODE REVIEW

File: ajax/comments.php [hunk 1]
```diff
@@ -81,7 +83,58 @@
...
+                        ]);
+                        $tmpname = [
+                            'comment' => $comment,
+                        ];
+                    }
+                }
+            }
+            echo($tmpname["comment"] ?? '');
 
             if (isset($_POST['withlink']) && isset($tmpname['link'])) {
```
Comment (AdrienClairembault) [inline ajax/comments.php:139]: `$tmpname['link']` is only set when id is 0. Is this normal ?

File: ajax/comments.php [hunk 2]
```diff
@@ -81,7 +83,58 @@
...
+            if ($_POST['value'] == 0) {
+                $tmpname = [
+                    'link'    => $CFG_GLPI['root_doc'] . "/front/group.php",
+                    'comment' => "",
+                ];
+            } else {
+                $group = new \Group();
+                if (is_array($_POST["value"])) {
+                    $comments = [];
+                    foreach ($_POST["value"] as $groups_id) {
```
Comment (AdrienClairembault) [inline ajax/comments.php:105]: I have trouble understanding how multiples values would be sent.
Isn't this a tooltip that is shown when you hover a group ?

File: templates/components/group/info_card.html.twig [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            </div>
+         {% endif %}
+      </div>
+   </div>
+
+   {% if can_edit %}
+      <div class="col">
+         <a class="btn btn-icon btn-sm btn-outline-secondary" href="{{ path('front/preference.php') }}"
+            title="{{ __('Edit') }}" data-bs-toggle="tooltip" data-bs-placement="right">
+            <i class="fas fa-group-edit"></i>
```
Comment (AdrienClairembault) [inline templates/components/group/info_card.html.twig:58]: Maybe we should use a tabler icon instead ?

File: templates/components/group/info_card.html.twig [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+               <i class="ti ti-notes"></i>
+               {{ group['comment'] }}
+            </div>
+         {% endif %}
+      </div>
+   </div>
+
+   {% if can_edit %}
+      <div class="col">
+         <a class="btn btn-icon btn-sm btn-outline-secondary" href="{{ path('front/preference.php') }}"
```
Comment (AdrienClairembault) [inline templates/components/group/info_card.html.twig:56]: Why is there a link to preferences in a group tooltip ? Am I missing something ?

**Review by AdrienClairembault — CHANGES_REQUESTED** [review]

File: templates/components/group/info_card.html.twig [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+<div class="p-0 group-info-card">
```
Comment (AdrienClairembault) [inline templates/components/group/info_card.html.twig:34]: It seems `group-info-card` is never used.

```suggestion
<div class="p-0">
```

**Review by AdrienClairembault — APPROVED** [review]

File: templates/components/group/info_card.html.twig (hunk 5 above)
Comment (Rom1-B) [reply templates/components/group/info_card.html.twig:34]: now it's useful :)

**Review by AdrienClairembault — APPROVED** [review]

**Review by trasher — APPROVED** [review]
LGTM

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**